### ISSUE TITLE
Using without pin ?

### ISSUE THREAD

**Issue by yaouenvoyage** [tttapa/Control-Surface#300]
Sorry for my stupid question and for my english, but I trying to use this wonderfull library without connected hardware just using variable that contains midi note or midi parameters or events. 
Is it possible and how to do that ?
Many thanks for help

**Comment by tttapa** [comment]
Yes, you can send pretty much any MIDI event using the [`MIDI_Sender`](https://tttapa.github.io/Control-Surface-doc/Doxygen/d6/d72/classMIDI__Sender.html) methods. The [MIDI-Output.ino](https://tttapa.github.io/Control-Surface-doc/Doxygen/d8/d9b/MIDI-Output_8ino-example.html) example shows how to use them.
